gobot/cmd/root.go

```diff
@@ -238,33 +239,42 @@ func receiveResults(ctx context.Context, redisHostPort string, logger *zap.Sugar
 	for {
 		select {
 		case <-ctx.Done():
+			logger.Info("Context cancelled, stopping receiveResults")
 			return
 		default:
-			count, err := r.LLen("results").Result()
+			count, err := r.LLen(ctx, redisQueueResults).Result()
 			if err != nil {
 				logger.Errorf("Redis Client Error: %v", err)
 				continue
 			}
 			if count == 0 {
+				logger.Info("No jobs in results, sleeping for 5 seconds")
```

Comment:
@nerdalert I think it's good to mark this debug, otherwise it's gonna fill the log every 5 second if bot is idle?